### ISSUE TITLE
When I run bcnn_DD_woft.py, my computer is laggy

### ISSUE THREAD

**Issue by yuqiu1233** [abhaydoke09/Bilinear-CNN-TensorFlow#35]
First , thanks for your contribution。 when I try to run  bcnn_DD_woft.py ，my computer is  too laggy to do anything. I have to stop the program . Do you know what cause this ? Is the file new_train_488.h5 too big? or video memory(11G) of my computer is not enough? How can I solve this problem?  I hope you can reply ,thank you!

**Comment by abhaydoke09** [comment]
Yes.. new_train_488.h5 is big.. that will take up lot of the memory. You can run it on a GPU with more memory. The I used had 22G memory. Or you can experiment with 224x224 size image. you'll have to make some changes in the network configuration, but it should work faster.
